src/components/tasks/CreateTask.jsx

```diff
@@ -0,0 +1,54 @@
+import { Fragment, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import sanitizeInput from "src/utils/helpers/sanitizeInput";
+import "src/components/tasks/CreateTask.component.scss";
+
+const CreateTask = ({ onAddTask }) => {
+  const [task, setTask] = useState("");
+  const inputRef = useRef(null);
+
+  function focusInputRef() {
+    inputRef.current.focus();
+  }
+
+  useEffect(focusInputRef, []);
+
+  function handleChange(event) {
+    setTask(event.target.value);
+  }
+
+  function handleAddTask() {
+    const sanitizedValue = sanitizeInput(task);
+    if (sanitizedValue) {
+      onAddTask(sanitizedValue);
+    }
+    setTask("");
+  }
+
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
```

Comment:
I would introduce a const module that will include all event's consts. 

Also do not use string directly.